Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Storage/Queries/FareParameters.hs

```diff
@@ -12,6 +12,8 @@
  the GNU Affero General Public License along with this program. If not, see <https://www.gnu.org/licenses/>.
 -}
 {-# OPTIONS_GHC -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-incomplete-patterns #-}
+{-# OPTIONS_GHC -Wno-deprecations #-}
```

Comment:
Please don't use these two pragmas, they are not safe. You can use `error "TODO"` instead of `undefined`, then pragma not required

Reply:
okay